content/general/concepts/unit-testing/unit-testing.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: 'Unit Testing'
+Description: 'Unit testing involves testing parts of code to verify its behavior.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Debugging'
+  - 'Development'
+CatalogContent:
+  - 'learn-javascript-unit-testing'
+---
+
+Unit testing is an essential part of the coding process. It verifies that all parts of the code work correctly.
+
+A part or unit of code can be anything from a class function to a small snippet of code that performs a specific task. Testing individual components makes it easier to find exactly where an error lies.
+
+There are usually two types of unit testing:
+
+- Manual testing
+- Automated testing
+
+## Manual Testing
+
+Manual testing is when tests are done directly by people. It can involve clicking on objects, providing input, or scrolling through items.
```

Comment:
```suggestion
Manual testing is when tests are done directly by developers. This can involve actions such as clicking on objects, providing input, or scrolling through items.
```

Reply:
I said "people" here because it doesn't have to be done by the developer specifically. It can be done by a user or client. Maybe there is another word that can be used other than "people" but not as restrictive as "developer". 🤔

Reply:
Good point! Thanks, Kyra! 😃

Reply:
I updated my suggestion to include "people" 👍